Sources/Core/Sources/GUI/GUISprite.swift

```diff
@@ -22,6 +22,7 @@ public enum GUISprite {
   case singleChestTopHalf
   case singleChestBottomHalf
   case doubleChest // No top/bottom half since it just uses the full image
```

Comment:
Feel free to rename these with their generic names instead. (also don't need the comment once the comment below is resolved cause it'll be clear which have halves and which don't).